tests/sceneManager_test.cpp

```diff
@@ -3,14 +3,64 @@
 #include "obj/objects.h"
 #include "gtest/gtest.h" // https://google.github.io/styleguide/cppguide.html#Namespaces
 
+// TODO: change underscore to the end
+
 using core::Ray;
 using core::RayHitData;
 using core::Vec3;
 using generators::PointSource;
 using objects::Object;
 using objects::TriangleObj;
 
-const double kSkipFreq = 1000;
 generators::FakeRandomGen fakeRand;
 
-// TODO: Change all doubles to float
+class SceneManagerTest : public ::testing::Test
+{
+protected:
+    SceneManager manager;
+    double energyCollectorRadius = objects::EnergyCollector(Vec3()).getRadius(); // TODO: I dont like this. Vec3 should be skip ;-)
```

Comment:
"Vec should be skip" seems like you are missing the point.

You are not supposed to create an object with fake data like this to get the radius out of it. Radius should be configurable, and it will be dependent on the number of collectors. So once you add the number of collectors to the constructor of SceneManager this will change.

Reply:
added to my TODO's list ; - )